Formula/bcftools.rb

```diff
@@ -20,7 +22,6 @@ class Bcftools < Formula
 
   depends_on "gsl"
   depends_on "htslib"
-  depends_on "xz"
```

Comment:
Does `brew linkage bcftools` show no linkage with `xz`?

Reply:
Indeed. The only homebrew libraries it lists are
```
Homebrew libraries:
  /usr/local/opt/gsl/lib/libgsl.25.dylib (gsl)
  /usr/local/opt/htslib/lib/libhts.3.dylib (htslib)
```